interhost.py

```diff
@@ -8,8 +8,16 @@
 
 import Bio.AlignIO
 from Bio import SeqIO
-import argparse, logging, os, array, bisect
-try :
+import argparse, logging, os, sys, array, bisect
+
+# try to import izip (in the case of Python 2). 
+# No alternate needed as Python3 zip() is iterator-based.
+try:
+    from itertools import izip
+except ImportError:
+    pass
+
+try:
```

Comment:
Sounds good. I generally try to avoid overwriting built-ins, your point is valid and I'll change it.